src/api/index.ts

```diff
@@ -1,8 +1,85 @@
 import 'reflect-metadata';
-import AppDataSource from './data-source';
+import './entities/user.js';
+import './entities/attempt.js';
+import './entities/test.js';
+import './entities/test-result.js';
+import AppDataSource from './data-source.js';
+import express, {
+  Request as ExpressRequest,
+  Response,
+  NextFunction,
+} from 'express';
+import session from 'express-session';
+import passport from 'passport';
+import flash from 'connect-flash';
+import authRouter, { seedAdminUser } from './routes/auth.routes.js';
+import attemptRouter from './routes/attempt.routes.js';
+import testRouter from './routes/test.routes.js';
+
+const app = express();
+app.set('trust proxy', 1);
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+// Session configuration
+const sessionSecret = process.env.JWT_SECRET || process.env.SESSION_SECRET;
+if (!sessionSecret) {
+  console.error(
+    'FATAL ERROR: JWT_SECRET or SESSION_SECRET environment variable for session is not set. Application will terminate.'
+  );
+  process.exit(1);
+}
+
+app.use(
```

Comment:
Save sessions to disk
https://github.com/klntsky/belief.garden/blob/65462e757542b6edc83d64a631e33bf4b058752a/src/app.js#L53